.github/workflows/test-ios-device.yaml

```diff
@@ -107,10 +107,12 @@ jobs:
           DEVICE_VERSION: ${{ matrix.os_version }}
         run: |
           set +e
-          output="$(./run_ios_testlab 2>&1 | tee /dev/tty)"
+          exec 3>&1
+          output="$(./run_ios_testlab 2>&1 | tee >(cat - >&3))"
```

Comment:
what is `3`?

Reply:
idk xdd I don't remember but it was needed for live streaming logs from `./run_ios_testlab` with a help of `| tee >(cat - >&3)`. I removed it because I think we don't need live streaming - they will be printed when te script is done.